driver/driver.go

```diff
@@ -23,77 +32,175 @@ type Driver interface {
 	Stats(api.ScannerStats) error
 }
 
-func Run(ctx context.Context, d Driver, program ast.Proc, zctx *zson.Context, reader zio.Reader, cfg Config) error {
-	ctx, cancel := context.WithCancel(ctx)
-	defer cancel()
+type Config struct {
+	Custom    kernel.Hook
+	Logger    *zap.Logger
+	Span      nano.Span
```

Comment:
I take it this will eventually become something like Range with some kind of expression that conveys min and max for a query?

Reply:
Oh I see: https://github.com/brimdata/zed/pull/2669/files#diff-abd1a262c5c1943d230a7d41ce0c7c1ea7dd417ff7e5c63a470ffcb884dce7b5R163

Doesn't look like this field is being used.

Reply:
Yes, thanks.  Removing Span from driver.Config.  I've been trying to get rid of driver.Config{} and it will soon be down to logger and statstick after we get rid of custom.  Then we can rip it out.